win32ss/gdi/ntgdi/freetype.c

```diff
@@ -2994,38 +2988,91 @@ static unsigned int get_bezier_glyph_outline(FT_Outline *outline, unsigned int b
     return needed;
 }
 
-static INT
-IntRequestFontSize(PDC dc, FT_Face face, LONG Width, LONG Height)
+static FT_Error
+IntRequestFontSize(PDC dc, PFONTGDI FontGDI, LONG lfWidth, LONG lfHeight)
 {
+    FT_Error error;
     FT_Size_RequestRec  req;
+    FT_Face face = FontGDI->SharedFace->Face;
+    TT_OS2 *pOS2;
+    TT_HoriHeader *pHori;
+    FT_WinFNT_HeaderRec WinFNT;
+    LONG A, D, AD;
```

Comment:
Maybe use slightly more human-readable variable names: Ascender, Descender, Total (??) (it is also OK to move these into the `if (lfHeight > 0)` block where you use them).

Reply:
Renamed. OK.